modules/eventing/pages/eventing-curl-spec.adoc

```diff
@@ -1,18 +1,25 @@
 = cURL
 :page-edition: Enterprise Edition
 
-The curl() function provides a way of interacting with external entities using HTTP.
+The curl() function provides a way of interacting with external entities via a REST endpoint using HTTP or HTTPS.
 
-Eventing functions can interact with external systems by using curl function to call their REST API. The possibility to interact with external systems opens a lot of new use cases, such as propagation of data changes to other systems, notifying the application about interesting events, enriching documents with data from external systems, and so on. This feature (introduced in 6.5) is both reliable and secure: the cURL calls are limited to a predefined set of URL bindings, for each binding we can specify authentication, encryption and certificate validation as necessary.  
+Eventing functions can interact with external systems by using curl function to call their REST API. The possibility to interact with external systems opens a lot of new use cases, such as
```

Comment:
```suggestion
Eventing functions can interact with external systems by using the curl() function to call their REST API. The possibility to interact with external systems opens a lot of new use cases, such as:
```

Trying to distinguish between `curl()` (the Eventing function) and `cURL` (the underlying technology).